mass_editing/tests/test_mass_editing.py

```diff
@@ -39,7 +55,23 @@ def _create_partner(self):
             'email': 'example@yourcompany.com',
             'phone': 123456,
             'category_id': [(6, 0, categ_ids)],
+            'notify_email': 'always'
+        })
+
+    def _create_partner_title(self):
+        """Create a Partner Title."""
+        # Loads German to work with translations
+        self.lang_model.load_lang('de_DE')
```

Comment:
Neat trick! It would be great if you open PR about this to maintainer-tools in either the module template or the docs 😊